MapboxNavigation/NavigationMapView.swift

```diff
@@ -1350,3 +1378,130 @@ extension NavigationMapView {
     }
 }
 
+// MARK: - Building Extrusion Highlights
+
+private struct BuildingHighlightAttributes {
+    var location: CLLocationCoordinate2D = kCLLocationCoordinate2DInvalid
+    var highlightColor: UIColor = .defaultBuildingHighlightColor
+    var identifier: UInt64? = nil
+}
+
+extension NavigationMapView {
+       
+    /**
+     Receives coordinates for searching the map for buildings. If buildings are found, they will be highlighted in 2d or 3d depending on the `in3D` value.
+     `in3D` defaults to true
+     `buildingHighlightingEnabled` must be set to true before this function will work.
+     */
+    public func highlightBuildings(for coordinates: [CLLocationCoordinate2D], in3D: Bool = true) {
+        let attributes = coordinates.map { BuildingHighlightAttributes(location: $0, highlightColor: buildingHighlightColor) }
+        highlightBuildings(with: attributes, in3D: in3D)
+    }
+    
+    /**
+     Removes the highlight from all buildings highlighted by `highlightBuildings(for coordinates:, in3D:)`.
+     */
+    public func unhighlightBuildings() {
+        highlightBuildings(nil)
+    }
+    
+    private func highlightBuildings(with attributes: [BuildingHighlightAttributes], in3D: Bool) {
```

Comment:
What is the motivation to keep `highlightBuildings(...)` methods family and `buildingId(coordinate:)` private? Those might be not as convenient as above, but it may be handy for users who need finer control over highlighting.

Reply:
I think in initial version it's better to keep API as simple as possible. In case if there is such need we'll expose additional methods.